### ISSUE TITLE
Istio Pilot Multiline output

### ISSUE THREAD

**Issue by jsenon** [istio/istio#6112]
Istio: 0.7.1
mTLS: disable

Stupid question, do we still need to use, multiline for logger:

```multierror "github.com/hashicorp/go-multierror```

https://github.com/istio/istio/blob/d762ec7d8bce983a64949de5932a185a2ae7930a/pilot/pkg/model/egress_rules.go#L176

We can have some issue when we need to scrap `\n`

```
2018-06-08T09:04:51.779645Z warn Rejected rules: 1 error occurred:


* rule egress-rule/istio-system/my-egress-rule is rejected since it conflicts rules [egress-rule/myrules/aws-s3-range
```

Docker spread log with different line (Output is different that previous one):

```
{"log":"[2018-06-08 11:55:50.106][20][info][config] src/envoy/utils/config.cc:50] v2 mixer client config: transport {\n","stream":"stderr","time":"2018-06-08T11:55:50.106928186Z"}
{"log":"  check_cluster: \"mixer_check_server\"\n","stream":"stderr","time":"2018-06-08T11:55:50.106945217Z"}
{"log":"  report_cluster: \"mixer_report_server\"\n","stream":"stderr","time":"2018-06-08T11:55:50.106952239Z"}
{"log":"}\n","stream":"stderr","time":"2018-06-08T11:55:50.106957292Z"}
{"log":"mixer_attributes {\n","stream":"stderr","time":"2018-06-08T11:55:50.106961994Z"}
{"log":"}\n","stream":"stderr","time":"2018-06-08T11:55:50.10696692Z"}
{"log":"forward_attributes {\n","stream":"stderr","time":"2018-06-08T11:55:50.106971103Z"}
{"log":"  attributes {\n","stream":"stderr","time":"2018-06-08T11:55:50.10697428Z"}
{"log":"    key: \"source.ip\"\n","stream":"stderr","time":"2018-06-08T11:55:50.106977724Z"}
{"log":"    value {\n","stream":"stderr","time":"2018-06-08T11:55:50.10698109Z"}
{"log":"      bytes_value: \"\\000\\000\\000\\000\\000\\000\\000\\000\\000\\000\\377\\377d`\\010\\334\"\n","stream":"stderr","time":"2018-06-08T11:55:50.106984175Z"}
{"log":"    }\n","stream":"stderr","time":"2018-06-08T11:55:50.106988247Z"}
{"log":"  }\n","stream":"stderr","time":"2018-06-08T11:55:50.10699124Z"}
{"log":"  attributes {\n","stream":"stderr","time":"2018-06-08T11:55:50.106994297Z"}
{"log":"    key: \"source.uid\"\n","stream":"stderr","time":"2018-06-08T11:55:50.106997353Z"}
{"log":"    value {\n","stream":"stderr","time":"2018-06-08T11:55:50.107000538Z"}
{"log":"      string_value: \"kubernetes://abcabc\"\n","stream":"stderr","time":"2018-06-08T11:55:50.107003472Z"}
{"log":"    }\n","stream":"stderr","time":"2018-06-08T11:55:50.107006785Z"}
{"log":"  }\n","stream":"stderr","time":"2018-06-08T11:55:50.107009738Z"}
{"log":"}\n","stream":"stderr","time":"2018-06-08T11:55:50.107012753Z"}
{"log":"\n","stream":"stderr","time":"2018-06-08T11:55:50.107015682Z"}
```

What's your feeling about that?

**Comment by alexburnos** [comment]
@jsenon could you please clarify, what exactly seems to be a problem? Do I understand correctly that you are saying that multi-line logs that are produced by go-multierror are hard to parse automatically?

**Comment by alexburnos** [comment]
Ok, go-multierror is used across Istio in general. To change it - it will be overall refactoring of logging approach. Not sure if it is feasible.

@andraxylia can you make a call here pls?
